client/js/render.js

```diff
@@ -305,18 +315,24 @@ function loadMoreHistory(entries) {
 	});
 }
 
-sidebar.on("click", ".collapse-network", (e) => {
-	const collapseButton = $(e.target);
+sidebar.on("click", "button.collapse-network", (e) => collapseNetwork($(e.target)));
 
+function collapseNetwork(target) {
+	const collapseButton = target.closest(".collapse-network");
+	const networks = JSON.parse(storage.get("thelounge.networks.collapsed")) || {};
+	const networkuuid = collapseButton.closest(".network").attr("data-uuid");
```

Comment:
`.data("uuid")`?

Reply:
`.data` doesn't refer to the `data-` attributes, does it? I thought it was completely separate to that.

Reply:
It fallbacks to `data-` if `.data()` wasn't changed before (in which case it uses internal cache).

Reply:
Huh, well, I did not know that. Alright, I shall change that up.